noxfile.py

```diff
@@ -105,7 +124,25 @@ def tests_brevitas_examples_cpu(session, pytorch, jit_status):
     install_pytorch(pytorch, session)
     install_torchvision(pytorch, session)  # For CV eval scripts
     session.install('--upgrade', '.[test, tts, stt, vision]')
-    session.run('pytest', '-n', 'logical', 'tests/brevitas_examples')
+    session.run(
+        'pytest',
+        '-n',
+        'logical',
+        '--ignore-glob',
```

Comment:
what's this ignore-glob?

Reply:
I'm using it to ignore `test_llm.py`. Otherwise the file gets loaded for the other tests and as a result `optimum-amd`, `transformers` and other packages become a dependency for all `brevitas_examples` tests.

Open to other options too, but this seemed like the best at the time.